packages/playground-examples/copy/ja/TypeScript/Primitives/Unknown and Never.ts

```diff
@@ -17,75 +17,73 @@ const myAccount = jsonParser(`{ "name": "Dorothea" }`);
 myAccount.name;
 myAccount.email;
 
-// If you hover on jsonParser, you can see that it has the
-// return type of any, so then does myAccount. It's possible
-// to fix this with Generics - but it's also possible to fix
-// this with unknown.
+// 変数jsonParserにカーソルを合わせると、戻り値の型がany型であることがわかります。
+// そして、変数myAccountも同様にany型になります。
+// こちらについて、ジェネリクスを使って型付けすることもできますが、
+// unknown型を使うことによっても修正できます。
 
 const jsonParserUnknown = (jsonString: string): unknown => JSON.parse(jsonString);
 
 const myOtherAccount = jsonParserUnknown(`{ "name": "Samuel" }`);
 
 myOtherAccount.name;
 
-// The object myOtherAccount cannot be used until the type has
-// been declared to TypeScript. This can be used to ensure
-// that API consumers think about their typing up-front:
+// myOtherAccountの型をTypeScriptに示すまでは、myOtherAccountのプロパティを使用する
+// ことができません。こうすることによって、APIの使用者に、型について前もって
+// 考えさせることができます。
 
 type User = { name: string };
 const myUserAccount = jsonParserUnknown(`{ "name": "Samuel" }`) as User;
 myUserAccount.name;
 
-// Unknown is a great tool, to understand it more read these:
+// unknown型はとても優れたツールです。
+// unknown型をより深く理解したい場合はこれらを参照してください。
```

Comment:
行数を変えるとdiffが大変そうなので揃えて訳してます。
```suggestion
// unknown型はとても優れたツールです。より深く理解したい場合はこれらを参照してください:
```